### PR TITLE
Update to Outdated files in the dev-1.18-ko.5 branch.

### LINKED ISSUE #21311: Outdated files in the dev-1.18-ko.5 branch
# This is a Bug Report
## Problem
Outdated files in the upstream/dev-1.18-ko.5 branch.

### 24 files to be modified 
1. [ ] content/en/docs/concepts/architecture/cloud-controller.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/architecture/nodes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/cluster-administration/cloud-providers.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/configuration/overview.md 1 file changed, 2 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/concepts/containers/images.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md 1 file changed, 4 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/overview/working-with-objects/namespaces.md 1 file changed, 5 insertions(+), 5 deletions(-)
1. [ ] content/en/docs/concepts/policy/pod-security-policy.md 1 file changed, 4 insertions(+)
1. [ ] content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md 1 file changed, 2 insertions(+), 13 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/connect-applications-service.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/dns-pod-service.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/statefulset.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/pods/init-containers.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/contribute/new-content/overview.md 1 file changed, 3 insertions(+)
1. [ ] content/en/docs/home/_index.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/setup/production-environment/tools/kubeadm/ha-topology.md 1 file changed, 5 insertions(+)
1. [ ] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/kustomization.md 1 file changed, 12 insertions(+), 13 deletions(-)
1. [ ] content/en/docs/tutorials/hello-minikube.md 1 file changed, 6 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tutorials/stateful-application/zookeeper.md 1 file changed, 1 deletion(-)
1. [ ] content/en/training/_index.html 1 file changed, 1 insertion(+), 1 deletion(-)

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `upstream/dev-1.18-ko.5` branch.

For example:
```
# checkout `upstream/dev-1.18-ko.5`
...
# check what is updated in the upstream 
git diff upstream/dev-1.18-ko.4 upstream/dev-1.18-ko.5 -- content/en/docs/concepts/architecture/cloud-controller.md
# apply changes to content/ko
vi content/ko/docs/concepts/architecture/cloud-controller.md
...
# commit and push
...
# make PR to `upstream/dev-1.18-ko.5`
```

## Pages to Update

/language ko

### PR DESCRIPTION
**Pull request by ysyukr**
Fix #21311

### 19 of 24 files to be modified
- [ ] content/en/docs/concepts/architecture/cloud-controller.md
> 변경점이 문서에는 영향이 없어 미반영
- [ ] content/en/docs/concepts/architecture/nodes.md
> 오타 수정으로 한글 문서에는 영향이 없어 미반영
- [x] content/en/docs/concepts/cluster-administration/cloud-providers.md
- [x] content/en/docs/concepts/configuration/overview.md
- [x] content/en/docs/concepts/containers/images.md
- [x] content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md
- [x] content/en/docs/concepts/overview/working-with-objects/namespaces.md
- [x] content/en/docs/concepts/policy/pod-security-policy.md
- [x] content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md
- [x] content/en/docs/concepts/services-networking/connect-applications-service.md
- [ ] content/en/docs/concepts/services-networking/dns-pod-service.md
> 한글 문서에는 반영되어있음
- [x] content/en/docs/concepts/workloads/controllers/daemonset.md
- [ ] content/en/docs/concepts/workloads/controllers/deployment.md
> 오타 수정으로 한글 문서에는 영향이 없어 미반영
- [x] content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md
- [] content/en/docs/concepts/workloads/controllers/statefulset.md
> #20970 에서 수정됨
- [x] content/en/docs/concepts/workloads/pods/init-containers.md
- [x] content/en/docs/contribute/new-content/overview.md
- [x] content/en/docs/home/_index.md
- [x] content/en/docs/setup/production-environment/tools/kubeadm/ha-topology.md
- [x] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md
- [x] content/en/docs/tasks/manage-kubernetes-objects/kustomization.md
- [x] content/en/docs/tutorials/hello-minikube.md
- [x] content/en/docs/tutorials/stateful-application/zookeeper.md
- [x] content/en/training/_index.html

/language ko

### CODE REVIEW

File: content/ko/docs/contribute/new-content/overview.md [hunk 1]
```diff
@@ -54,5 +54,8 @@ CLA에 서명하지 않은 기여자의 풀 리퀘스트(pull request)는 자동
 
 PR 당 하나의 언어로 풀 리퀘스트를 제한한다. 여러 언어로 동일한 코드 샘플을 동일하게 변경해야 하는 경우 각 언어마다 별도의 PR을 연다.
 
+## 기여자를 위한 도구들
+
+`kubernetes/website` 리포지터리의 [문서 기여자를 위한 도구](https://github.com/kubernetes/website/tree/master/content/en/docs/doc-contributor-tools) 디렉터리에는 기여 여정이 좀 더 순조롭게 할수 있도록 도와주는 도구들이 포함되어 있다.
```
Comment (pjhwa) [inline content/ko/docs/contribute/new-content/overview.md:59]:
```suggestion
`kubernetes/website` 리포지터리의 [문서 기여자를 위한 도구](https://github.com/kubernetes/website/tree/master/content/en/docs/doc-contributor-tools) 디렉터리에는 기여 여정이 좀 더 순조롭게 진행되도록 도와주는 도구들이 포함되어 있다.
```

- 좀 더 자연스럽게 바꿔봤습니다.

File: content/ko/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 2]
```diff
@@ -339,11 +339,11 @@ resources:
 EOF
 ```
 
-`kubectl kustomize ./`의 리소스는 디플로이먼트와 서비스 오브젝트 모두를 포함한다.
+`kubectl kustomize ./`의 리소스는 디플로이먼트와 서비스 오브젝트가 모두 포함되어 있다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/manage-kubernetes-objects/kustomization.md:342]:
```suggestion
`kubectl kustomize ./`의 리소스에는 디플로이먼트와 서비스 오브젝트가 모두 포함되어 있다.
```

- 좀 더 자연스럽게 바꿔봤습니다.

File: content/ko/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -47,7 +47,9 @@ Katacode는 무료로 브라우저에서 쿠버네티스 환경을 제공한다.
 
     {{< kat-button >}}
 
-    {{< note >}}Minikube를 로컬에 설치했다면 `minikube start`을 실행한다.{{< /note >}}
+{{< note >}}
+    Minikube를 로컬에 설치했다면 `minikube start`을 실행한다.
```
Comment (pjhwa) [inline content/ko/docs/tutorials/hello-minikube.md:51]:
```suggestion
    Minikube를 로컬에 설치했다면 `minikube start`를 실행한다.
```

- 조사를 바꿨습니다.

File: content/ko/docs/tutorials/hello-minikube.md [hunk 4]
```diff
@@ -112,7 +114,9 @@ Katacode는 무료로 브라우저에서 쿠버네티스 환경을 제공한다.
     kubectl config view
     ```
 
-    {{< note >}}`kubectl` 명령어에 관해 자세히 알기 원하면 [kubectl 개관](/docs/user-guide/kubectl-overview/)을 살펴보자.{{< /note >}}
+{{< note >}}
+    `kubectl` 명령어에 관해 자세히 알기 원하면 [kubectl 개관](/docs/user-guide/kubectl-overview/)을 살펴보자.
```
Comment (pjhwa) [inline content/ko/docs/tutorials/hello-minikube.md:118]:
```suggestion
    `kubectl` 명령어에 관해 자세히 알기 원하면 [kubectl 개요](/docs/user-guide/kubectl-overview/)을 살펴보자.
```

- '개요'가 좀 더 친숙할 것 같아서 바꿔봤습니다.

**Comment by pjhwa** [comment]
다시 확인하였습니다.
@ysyukr 수고 많으셨습니다.

/lgtm

**Comment by gochist** [comment]
@ysyukr @pjhwa 감사합니다!
/lgtm
/approve

**Pull request merged**